tt_metal/impl/dispatch/dispatch_address_map.hpp

```diff
@@ -14,6 +14,7 @@ static constexpr uint32_t PCIE_ALIGNMENT = NOC_PCIE_READ_ALIGNMENT_BYTES >= NOC_
 
 // Command queue pointers
 constexpr static uint32_t CQ_PREFETCH_Q_RD_PTR = L1_UNRESERVED_BASE;
+constexpr static uint32_t CQ_PREFETCH_Q_PCIE_RD_PTR = CQ_PREFETCH_Q_RD_PTR + sizeof(uint32_t); // Used to notify host of how far device has gotten.
```

Comment:
maybe add a comment explaining why this does not have to l1 aligned because cq_prefetch just locally updates this address

Reply:
Added a comment for this in latest commit